cves/kernel/CVE-2019-13648.yml

```
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: | 
+  On PowerPC Machines, local users can trick the system into attempting to use 
+  an unsupported/disabled feature which causes a system crash. The user can set 
+  bits of the MSR(Machine-State Register) to allow HTM(Hardware Transactional Memory), even 
+  when that feature is disabled or unsupported by the machine. This causes the system to crash, 
+  thus classifying as a Denial of Service vulnerability.
```

Just some slight rewording/clarifications to consider:

- Usually you spell out the full term and then abbreviate in parenthesis.
- In the fix commit, https://github.com/torvalds/linux/commit/f16d80b75a096c52354c6e0a574993f3b0dfbdfe, they use TM not HTM.

```suggestion
description: |
  On certain PowerPC Machines, that lack Transactional Memory (TM), local users
  can trick the system into attempting to use TM which causes a system crash.
  Users can set the TM bits of the Machine-State Register (MSR), even when TM is
  not supported by the system. This causes the system to crash, thus classifying
  it as a Denial of Service (DoS) vulnerability.
```

Optionally, you could expand on the description a little more:

- What is MSR?
- What is Transactional Memory?
- Maybe provide a little more context.

---

Fixed